src/lib.rs

```diff
@@ -389,44 +364,39 @@ mod tls {
                 }
             });
 
-            #[cfg(feature = "tracing")]
-            {
-                tracing::debug!("Certificates loaded (Count: {})", certs.len());
-            }
-
             Ok(CertVerifier { certs: certs })
         }
     }
 
-    impl rustls::ServerCertVerifier for CertVerifier {
+    impl ServerCertVerifier for CertVerifier {
```

Comment:
**thought:** I don't like having these custom `CertVerifier`. How do we know we didn't miss something important here?

Reply:
I agree we can try remove the custom verifier, but I'd say it's out of scope for this version bump.

I explored remove the custom cert verifier as part of this bump but hit a dead end with that because the default verifier flags the LND cert as invalid. It errors because cert is used as an end entity while it's not.

Reply:
It may be nice to have a custom verifier that defaults to the regular verifier and catches that specific case. Not sure how approachable that would be though

Reply:
Yeah, let's explore custom validation changes as follow up improvement